scripts/deploy_token.ts

```diff
@@ -0,0 +1,77 @@
+import 'dotenv/config'
```

Comment:
Why you added this file? We have **uploadAndInitToken** in deploy_core script

Reply:
**uploadAndInitToken** is used to deploy the ASTRO token, whereas this file - for an arbitrary token for further arbitrary pool deployment